nilearn/regions/region_extractor.py

```diff
@@ -67,6 +68,61 @@ def _threshold_maps_ratio(maps_img, threshold):
     return threshold_maps_img
 
 
+def _remove_small_regions(input_data, mask_data, index,
+                          affine, min_size):
```

Comment:
Kamalakar: could you please integrate this function: inline the code in the caller and remove it.